packages/replay-next/src/suspense/BuildIdCache.ts

```diff
@@ -51,17 +85,23 @@ export const recordingTargetCache = createSingleEntryCache<
   },
 });
 
-function getRecordingCapabilities(buildId: string) {
-  const recordingTarget = getRecordingTarget(buildId);
+function getRecordingCapabilities(
+  recordingTarget: RecordingTarget,
+  buildComponents: BuildComponents
+) {
   switch (recordingTarget) {
     case "chromium": {
+      const buildDate = buildDateStringToDate(buildComponents?.date ?? "");
+      const supportsObjectIdLookupsInEvaluations = buildDate >= new Date("2023-09-16");
```

Comment:
cc @Domiii for another example of how a capabilities API would be useful and the types of things FE is having to do to work around the absence of one 🙇🏼

Reply:
@bvaughn - Thanks! I added it [here](https://linear.app/replay/issue/RUN-1148/add-capabilities-metadata-api#comment-14401bf1)